be.telenet.user.QSSTV.yml

```diff
@@ -0,0 +1,69 @@
+app-id: be.telenet.user.QSSTV
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qsstv
+rename-desktop-file: qsstv.desktop
+rename-icon: qsstv
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+  - /bin/usr
+modules:
+  - name: hamlib
+    buildsystem: simple
```

Comment:
what's wrong with using `buildsystem: autotools` here?

Reply:
Thanks. I have changed it to `buildsystem: cmake-ninja`.